source/request-as-event-emitter.ts

```diff
@@ -70,11 +76,13 @@ export default (options, input?: TransformStream) => {
 		}
 
 		/* istanbul ignore next: electron.net is broken */
+		// No point in typing process.versions correctly, as
+		// process.version.electron is used only once, right here.
 		if (options.useElectronNet && (process.versions as any).electron) {
 			// @ts-ignore
 			const r = ({x: require})['yx'.slice(1)]; // Trick webpack
 			const electron = r('electron');
-			fn = electron.net || electron.remote.net;
+			requestFn = electron.net || electron.remote.net;
```

Comment:
Are there tests for this code? I realize this was not accessing `electron.net.request` as it should.

Reply:
No, too much overhead to include Electron testing to be worth it. We do manual test after doing changes to that code.